cvpysdk/subclient.py

```diff
@@ -169,6 +169,40 @@ def _get_subclients(self):
             response_string = self._commcell_object._update_response_(response.text)
             raise SDKException('Response', '101', response_string)
 
+    def _set_subclient_content_(self, subclient_content):
+        """Creates the list of content JSON to pass to the API to add a new Subclient
+            with the content passed in subclient content.
+
+            Args:
+                subclient_content (list)  --  list of the content to add to the subclient
+
+            Returns:
+                list - list of the appropriate JSON for an agent to send to the POST Subclient API
+        """
+        content = []
+
+        if self._backupset_object._agent_object.agent_name == "file system":
+            for path in subclient_content:
+                file_system_dict = {
+                    "path": path
+                }
+
+                content.append(file_system_dict)
+        elif self._backupset_object._agent_object.agent_name == "cloud apps":

```

Comment:
Sure Karthik, Please send me the design so I can make changes accordingly. Meanwhile I will get in touch with Jatin.Thanks!